README.md

```diff
@@ -342,17 +342,19 @@ larger limits on the amount of destination points.
 * country: ProtoCountry - Return the results that are within the specified country.
 * one_to_many: boolean - if one_to_many is equal to true, then it'll be a forward search (one to many matrix), false -
   backward search (many to one matrix). Default value is True.
+* properties: List[PropertyProto] - specifies which extra properties should be calculated in the response. 
 
 #### Returns:
 
-* travel_times: List[int] - The responses are in the form of a list where each position denotes either a travel time (in
-  seconds) of a journey, or if negative that the journey from the origin to the destination point is impossible.
+* results: TimeFilterProtoResponse - The response contains 2 lists, firstly travel times, where each position denotes either a travel time (in
```

Comment:
Probably contains 1 or 2 lists?

I would write it like this:

results: TimeFilterProtoResponse - The response contains:
* list of travel times, where each position denotes either a travel time (in seconds) of a journey, or if travel time is negative, that the journey from the origin to the destination point is impossible.
* (optional) list of distances where each position denotes distance (in meters) to the specified location.

Reply:
Thanks, I was struggling to word this properly. Implemented.